pkg/destroy/ovirt/destroyer.go

```diff
@@ -117,7 +117,7 @@ func (uninstaller *ClusterUninstaller) removeVM(vmsService *ovirtsdk.VmsService,
 func (uninstaller *ClusterUninstaller) removeTemplate(con *ovirtsdk.Connection) error {
 	if uninstaller.Metadata.Ovirt.RemoveTemplate {
 		search, err := con.SystemService().TemplatesService().
-			List().Search(fmt.Sprintf("name=%s", uninstaller.Metadata.InfraID)).Send()
+			List().Search(fmt.Sprintf("name=%s*", uninstaller.Metadata.InfraID)).Send()
```

Comment:
Probably the '*' wildcard could be substituted by the '-rhcos' string, 'cause I guess that ovirt is using the openstack [GenerateOpenStackImageName](https://github.com/openshift/installer/blob/85092be98f2e913e3ea9b45622b73c3bc44d6e98/pkg/rhcos/openstack.go#L47-L58) to get the image name.
Anyway I cannot imagine a problem using the '*' wildcard after the InfraID in the templates section, so I'll leave it as it is.

Reply:
there is no need for `*` since the engine search term is already a wildcard search - see the comment on line 126.